docs/tutorial_tips_tricks.rst

```diff
@@ -583,34 +583,3 @@ The following figure shows one part of the automatically generated pipeline for
 .. note::
 
    The ReFrame executable must be available in the Gitlab runner that will run the CI jobs.
-
-
-Skip tests based on the version of ReFrame
-------------------------------------------
-
-.. versionadded:: 3.7.0
-
-When writing a test you can specify the versions of ReFrame that are compatible with it like this:
-
-.. code-block:: python
-
-   class BaseFoo(rfm.RegressionTest, required_version=['x.x.x', '>y.y.y']):
-     ...
-
-   @rfm.simple_test
-   class Foo(BaseFoo):
-     ...
-
-
```

Comment:
I would still leave this part (not the detailed description below) but make it more concrete, for example use `>3.7.0' as a version string. And then point to the documentation.

Reply:
Oh, ok, ignore this comment. I thought that we had this in master.